src/rofiles-fuse/main.c

```diff
@@ -345,6 +345,26 @@ callback_create(const char *path, mode_t mode, struct fuse_file_info *finfo)
 }
 
 static int
+callback_read_buf (const char *path, struct fuse_bufvec **bufp,
+                   size_t size, off_t offset, struct fuse_file_info *finfo)
+{
+  struct fuse_bufvec *src;
+
+  src = malloc (sizeof (struct fuse_bufvec));
```

Comment:
Did fuse really add an interface for this that forces us to malloc for every read?   Reading the source, I guess so, but man, that's ugly.